scanpipe/pipelines/deploy_to_develop.py

```diff
@@ -259,6 +261,20 @@ def map_jar_to_grammar_source(self):
             project=self.project, jvm_lang=jvm.GrammarLanguage, logger=self.log
         )
 
+    @optional_step("Xtend")
```

Comment:
Should extend be an optional step or be part of all JVM based languages like Scala, Java and Kotlin ?

Reply:
Yes, they are. Coz they all are independent of each other. We won't run the same thing for Scala that we are running for Java. But if we are running same things then it wouldn't have been a separate optional step.

Reply:
Here is the background.

Xtend is a flexible and expressive dialect of Java (https://eclipse.dev/Xtext/xtend/)
It compiles `.xtend` files into readable `.java` source code, which is then compiled into JVM bytecode (`.class` files).
Xtend uses a source-to-source compiler, meaning it translates Xtend code into Java before passing it to a standard Java compiler.

There are couple ways to compile `.xtend` such as

Maven
-------
```
mvn compile
```

OR

Gradle
------
```
gradle build
```

Xtend plugin will generate `.java` files for the above

OR

Eclipse IDE
------------
Just save the .xtend file (Eclipse automatically generates the corresponding .java file)